src/utils/cplay.c

```diff
@@ -564,6 +671,30 @@ void get_codec_iec(FILE *file, struct compr_config *config,
 	codec->format = 0;
 }
 
+static int check_stdin(struct compress *compress)
+{
+	switch(do_pause()) {
+		case DO_PAUSE:
+			if (compress_pause(compress) != 0) {
+				fprintf(stderr, "Pause ERROR\n");
+				return -1;
+			}
+			is_paused = true;
+			break;
+		case DO_RESUME:
+			if (compress_resume(compress) != 0) {
```

Comment:
compress_pause_release

Reply:
I think this should be addressed in another series of patches as this one doesn't actually introduce `compress_resume()` but it just uses it.

Reply:
Agreed, this should be done as a different change